app/src/main/res/values/styles.xml

```diff
@@ -16,23 +16,31 @@
 
 <resources>
 
-    <!-- Base application theme. -->
-    <style name="BaseAppTheme" parent="Theme.MaterialComponents.Light.DarkActionBar">
-        <!-- Customize your theme here. -->
+    <!-- To see how these attributes are used, please visit https://material.io/develop/android/theming/color/ -->
+    <style name="Base.Theme.Sunflower" parent="Theme.MaterialComponents.DayNight.NoActionBar">
         <item name="colorPrimary">@color/colorPrimary</item>
-        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
+        <item name="colorOnPrimary">@color/colorOnPrimary</item>
+        <item name="colorPrimaryDark">@color/colorTabIconStroke</item>
+        <item name="colorOnSurface">@color/colorOnSurface</item>
         <item name="colorAccent">@color/colorAccent</item>
+        <item name="colorSurface">@color/colorSurface</item>
+        <item name="colorSecondary">@color/colorSecondary</item>
+        <item name="android:colorBackground">@color/colorBackground</item>
     </style>
 
-    <style name="AppTheme" parent="BaseAppTheme"/>
+    <style name="Theme.Sunflower" parent="Base.Theme.Sunflower"/>
 
-    <style name="AppTheme.NoActionBar">
+    <style name="Theme.Sunflower.NoActionBar">
         <item name="windowActionBar">false</item>
         <item name="windowNoTitle">true</item>
     </style>
 
-    <style name="AppTheme.AppBarOverlay" parent="ThemeOverlay.MaterialComponents.Dark.ActionBar"/>
+    <style name="Theme.Sunflower.AppBarOverlay" parent="ThemeOverlay.MaterialComponents.Dark.ActionBar"/>
 
-    <style name="AppTheme.PopupOverlay" parent="ThemeOverlay.MaterialComponents.Light"/>
+    <style name="Theme.Sunflower.PopupOverlay" parent="ThemeOverlay.MaterialComponents.Light"/>
```

Comment:
Apparently and after the refactor the style: _Theme.Sunflower.PopupOverlay_ appears as unused, maybe we should to remove it or add to the property visual element

Reply:
Great catch, thanks! Removed.